packages/misc/src/lib/misc.ts

```diff
@@ -353,7 +339,7 @@ export const logErrorWithRequestId = (id: string, ...args: any) => {
     globalThis.logManager.get(globalThis.logger.category, id).error(...args);
 };
 
-export const logError = (...args: any) => {
```

Comment:
Why is it OK to leave the check on L370 below here, but we had to remove it in `logErrorWithRequestId` and `log`?